Frameworks/CoreText/CTLine.mm

```diff
@@ -38,7 +44,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     ret->_ascent = _ascent;
     ret->_descent = _descent;
     ret->_leading = _leading;
-    ret->_runs = [_runs copy];
```

Comment:
copy by default doesn't copy the items, so each run would be sharing a reference to a string, which we would lose should the original run be dealloced

---
In reply to: [78260653](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78260653) [](ancestors = 78260653,78225046)